src/Microsoft.TestPlatform.AdapterUtilities/Resources/xlf/Resources.xlf

```diff
@@ -36,12 +36,12 @@ Example: 'System.Reflection.MethodBase' is not implemented on this platform!</no
       <trans-unit id="ErrorInvalidSequenceAt">
```

Comment:
I did a rebuild but I can't get the other xlf files to be updated, is there any specific command to run?

Reply:
Just build.cmd should do it.

Reply:
@Haplois  I've built this locally as well and I don't get updates in the localization files either. Any tips?

Reply:
We only update main resource file and the xlf file. All others are updated by the bots.